### PR TITLE
#301 Testing key in getLabel2 of jFormsDaoDatasource 

### PR DESCRIPTION
**Pull request by migratis**
Testing key provided to getLabel2 of jFormsDaoDatasource class if it is null or empty it returns null

### CODE REVIEW

File: lib/jelix/forms/jFormsDatasource.class.php [hunk 1]
```diff
@@ -177,6 +177,9 @@ public function setCriteriaControls($criteriaFrom = null){
  */
 class jFormsDaoDatasource extends jFormsDynamicDatasource {
 
+    if ($key === null || $key == "")
+        return null;
+    
     protected $selector;
```
Comment (laurentj) [inline lib/jelix/forms/jFormsDatasource.class.php]: Sorry, but you made the change oustide a method, in the class block...

Comment (migratis) [reply lib/jelix/forms/jFormsDatasource.class.php]: Oups ! I will make the correction as soon as possible...
I'm very sorry

**Comment by migratis** [comment]
ok correction done

**Comment by laurentj** [comment]
Thank you

**Pull request merged**